examples/reference-entity-model/loadData/CustomerLoadJSON.loadData.json

```diff
@@ -0,0 +1 @@
+{"name":"CustomerLoadJSON","description":"","sourceFormat":"json","targetFormat":"json","outputURIReplacement":".*CustomerLoadJSON,'/customer'","inputFilePath":"data-sets/CustomerLoadJSON"}
```

Comment:
I'd like to encourage a naming practice for steps - which are processes - that are verb-oriented instead of a set of nouns. So let's do "loadCustomerJSON" and "mapCustomerJSON". I think customers should follow the same practice, so we should demonstrate that in our examples. 

Even better would be to have names for the datasets other than "JSON" and "XML", as in the real world, users are likely to have e.g. "loadAbcCustomers" and "loadXyzCustomers". I think we can go with JSON and XML for now, but ideally we can have real data source names here and not just use data formats.

Reply:
I agree with all the suggestions Anyone from the team can later enhance this to be standardized. For now this is just aimed at getting a project together for upcoming demo that can be run in an instance that is not localhost as we had discussed in Friday's standup.